sql/full-text-search/varchar/query-v2/partitioning/many-partition.sql

```diff
@@ -0,0 +1,131 @@
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_20_04 PARTITION OF infection_numbers_influenza FOR VALUES IN ('20-04') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_21_01 PARTITION OF infection_numbers_influenza FOR VALUES IN ('21-01') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_21_02 PARTITION OF infection_numbers_influenza FOR VALUES IN ('21-02') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_21_03 PARTITION OF infection_numbers_influenza FOR VALUES IN ('21-03') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_21_04 PARTITION OF infection_numbers_influenza FOR VALUES IN ('21-04') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_22_01 PARTITION OF infection_numbers_influenza FOR VALUES IN ('22-01') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_22_02 PARTITION OF infection_numbers_influenza FOR VALUES IN ('22-02') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_22_03 PARTITION OF infection_numbers_influenza FOR VALUES IN ('22-03') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
+);
+
+CREATE TABLE infection_numbers_influenza_22_04 PARTITION OF infection_numbers_influenza FOR VALUES IN ('22-04') (
+    country varchar(16) NOT NULL,
+    prefecture varchar(4) NOT NULL,
+    city varchar(3) NOT NULL,
+    n_infection varchar(205),
+    city_code varchar(5) NOT NULL,
+    remarks varchar(200)
```

Comment:
Do we still need these column definitions?

Reply:
I removed these column definitions in https://github.com/pgroonga/pgroonga/pull/236/commits/c941e20463aea537b51131e2f0036d3f9f6df09c.